api/src/api/resource.js

```diff
@@ -42,9 +42,16 @@ router.get(
 
     if (requireLatLong) {
       const apiLatLong =
-        `https://www.mapquestapi.com/geocoding/v1/address?key=` +
-        `${process.env.MAPBOX_KEY}&maxResults=5&location=${resource.address},${resource.city},${resource.state},${resource.zip}`;
+        resource.address.length > 0 ||
```

Comment:
Is this logic just checking that at lest one of them is filled? or should all of them be filled

Reply:
It's checking if at least one is filled